fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -356,6 +358,8 @@ public Response createOrReplaceObjectRdf(
                 }
                 throw new ClientErrorException("Invalid Content Type " + requestContentType, UNSUPPORTED_MEDIA_TYPE);
             }
+        } catch (final Exception e) {
```

Comment:
@dbernstein is exactly correct here -- we don't want to introduce a JCR dependency here. Since the `BinaryStoreException` inherits from `jcr.RepositoryException`, which inherits from `java.lang.Exception`, I think the `catch (Exception)` will need to stay.